shared/teams/team/rows/invite-row/request/index.tsx

```diff
@@ -106,7 +106,7 @@ const TeamRequestRowNew = (props: Props) => {
       closeOnSelect={true}
       attachTo={attachTo}
       position="bottom left"
-      positionFallbacks={['top left']}
+      positionFallbacks={['center left', 'top left'] as const}
```

Comment:
might need to do `['center left' as const, 'top left' as const] as const`, or play with it